api/http/handler/edgestacks/edgestack_status_update.go

```diff
@@ -74,8 +74,24 @@ func (handler *Handler) edgeStackStatusUpdate(w http.ResponseWriter, r *http.Req
 	var stack portainer.EdgeStack
 
 	err = handler.DataStore.EdgeStack().UpdateEdgeStackFunc(portainer.EdgeStackID(stackID), func(edgeStack *portainer.EdgeStack) {
+		details := edgeStack.Status[payload.EndpointID].Details
+		details.Pending = false
+
+		switch *payload.Status {
+		case portainer.EdgeStackStatusOk:
+			details.Ok = true
+		case portainer.EdgeStackStatusError:
+			details.Error = true
+		case portainer.EdgeStackStatusAcknowledged:
+			details.Acknowledged = true
+		case portainer.EdgeStackStatusRemove:
+			details.Remove = true
+		case portainer.EdgeStackStatusImagesPulled:
+			details.ImagesPulled = true
+		}
```

Comment:
Is it intentional to exclude `case EdgeStackStatusRemoteUpdateSuccess:` ?

Reply:
yes, it's intended. EdgeStackStatusRemoteUpdateSuccess is used at the portainer side only. Agent never set this status.